client/lib/gsuite/is-google-monthly-product.ts

```diff
@@ -0,0 +1,6 @@
+import { GOOGLE_WORKSPACE_BUSINESS_STARTER_MONTHLY } from '@automattic/calypso-products';
+import type { ProductListItem } from 'calypso/state/products-list/selectors/get-products-list';
+
+export function isGoogleMonthlyProduct( googleMailProduct: ProductListItem ): boolean {
+	return googleMailProduct?.product_slug === GOOGLE_WORKSPACE_BUSINESS_STARTER_MONTHLY;
+}
```

Comment:
I don't think we want this file in this PR - we're not introducing the monthly product yet.